### ISSUE TITLE
Accessibility issues with HTML publishing of "Backlinks" and "Children"

### ISSUE THREAD

**Issue by kap42** [dendronhq/dendron#3369]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [ ] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [ ] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [x] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

The headings for "Backlinks" and "Children" in the HTML output are rendered as "<strong>", but for accessibility reasons, they should use headings. This is important for things like screen readers, where the headings are used for quick navigation on a page.

## To Reproduce

1. Go to any documentation page that have backlinks and/or children.

## Expected behavior

There should be headings instead of plain <strong>.

## Desktop (please complete the following information)

- OS: Windows 11
- Browser: Chrome
- Versions (all that apply to bug):
  - Dendron Extension: 0.106.0
  - VS Code version: 1.69.2
  - Dendron CLI: 0.106.0

## Dendron Log file

Please attach the output of `> Dendron:Dev: Open Logs` here
[dendron.log](https://github.com/dendronhq/dendron/files/9297418/dendron.log)

**Comment by hikchoi** [comment]
Thank you for the report. We'll keep track of this and let you know when this gets resolved.

**Comment by Harshita-mindfire** [comment]
This was a breaking change introduced in Dendron v0.76: https://wiki.dendron.so/notes/MQpi87qLnzKK5iK3FlxUp/#breaking-changes

**Comment by namjul** [comment]
Do we have a papertrail to the source of that change made in v0.76. It seems the idea behind that change was to remove it from the toc section. So maybe this issue is about replacing back to headings but not getting shown inside the toc section.

**Comment by kap42** [comment]
If headings aren't a good fit, then you might use ARIA's "role=heading". See: https://www.w3.org/TR/WCAG20-TECHS/ARIA12.html
